src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/textblockgooglestyleformatting/Example3.java

```diff
@@ -0,0 +1,58 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="TextBlockGoogleStyleFormatting"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.textblockgooglestyleformatting;
+// xdoc section -- start
+public class Example3 {
+  public String testMethod1() {
+    final String simpleScript1Violate =
+            """
+            this is simple test""";
+    // 2 violations above:
+    //   'Closing quotes (""") of text-block must be on the new line.'
+    //   'Text-block quotes are not vertically aligned'
+    final String simpleScript1Correct =
```

Comment:
please add empty line before each "correct" to visually separate pairs of Violate+Correct.
please apply to all other places.